CMakeLists.txt

```diff
@@ -442,7 +439,7 @@ while(library_variants_copy)
             DEPENDS clang lld llvm-ar llvm-config llvm-nm llvm-ranlib llvm-strip
             CONFIGURE_COMMAND ${MESON_EXECUTABLE} -Dincludedir=include -Dlibdir=lib --prefix <INSTALL_DIR> --cross-file <BINARY_DIR>/meson-cross-build.txt ${picolibc_SOURCE_DIR}
             BUILD_COMMAND ninja
-            INSTALL_COMMAND ${MESON_EXECUTABLE} install ${MESON_INSTALL_QUIET}
+            INSTALL_COMMAND meson install ${MESON_INSTALL_QUIET}
```

Comment:
Change this back. You can specify MESON_EXECUTABLE with a -D argument if cmake can't find meson automatically.

Reply:
Indeed merge issue.